src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -386,7 +414,16 @@ public static PreferenceSettings Load(string filePath)
             settings.CustomPackageFolders = settings.CustomPackageFolders.Distinct().ToList();
 
             return settings;
-		}
+        }
+
+        /// <summary>
+        /// Returns the static Python template file path
```

Comment:
since this is the public entry point for most use cases lets add some description of what this file at this path is used for.

Reply:
Good point, wasn't being descriptive enough 👍 . Added now, have a look please 👁